elasticapm/metrics/cpu_linux.py

```diff
@@ -0,0 +1,81 @@
+import multiprocessing
+import os
+import re
+import resource
+import threading
+
+from elasticapm.metrics.base_metrics import MetricsSet
+
+SYS_STATS = "/proc/stat"
+MEM_STATS = "/proc/meminfo"
+PROC_STATS = "/proc/self/stat"
+
+whitespace_re = re.compile(r"\s+")
+
+
+if not os.path.exists(SYS_STATS):
+    raise ImportError("This metric set is only available on Linux")
+
+
+class CPUMetricSet(MetricsSet):
+    def __init__(self, sys_stats_file=SYS_STATS, process_stats_file=PROC_STATS, memory_stats_file=MEM_STATS):
+        self.page_size = resource.getpagesize()
+        self.previous = {}
+        self._read_data_lock = threading.Lock()
+        self.sys_stats_file = sys_stats_file
+        self.process_stats_file = process_stats_file
+        self.memory_stats_file = memory_stats_file
+        with self._read_data_lock:
+            self.previous.update(self.read_process_stats())
+            self.previous.update(self.read_system_stats())
+        super(CPUMetricSet, self).__init__()
+
+    def collect(self):
+        cpu_count = float(multiprocessing.cpu_count())
```

Comment:
You don't need to divide by `cpu_count`. You're reporting the fraction of non-idle total CPU time, which is going to be in the range 0-1 already.

Reply:
fixed in 3048596